### ISSUE TITLE
"SQL Query as a service" as a webservice

### ISSUE THREAD

**Issue by tobiasgies** [engelsystem/engelsystem#1433]
From a discussion with @xuwhite:

* Build up a "library" of commonly used SQL queries that seem to be useful or necessary for users.
* Allow users that have a certain permission level (maybe bureaucrats and up) to fill in a few parameters and execute them
* Output results as a simple HTML table (for SELECTs) or number of affected rows (for INSERT / UPDATE).
* Ideally, build up metrics / telemetry about which of these queries are used often, and turn them into user-friendly features.

**Comment by MyIgel** [comment]
I don't think that having a "ugly hacks (TM)" page would be a great idea (maintenance, abuse, documentation etc.) but we definitely should have some "list of useful queries" / "features to be implemented" / query database thing
